maskrcnn_benchmark/config/paths_catalog.py

```diff
@@ -21,18 +20,38 @@ class DatasetCatalog(object):
             "coco/val2014",
             "coco/annotations/instances_valminusminival2014.json",
         ),
-        "voc_2007_trainval": ("voc/VOC2007", 'trainval'),
-        "voc_2007_test": ("voc/VOC2007", 'test'),
-        "voc_2012_train": ("voc/VOC2012", 'train'),
-        "voc_2012_trainval": ("voc/VOC2012", 'trainval'),
-        "voc_2012_val": ("voc/VOC2012", 'val'),
-        "voc_2012_test": ("voc/VOC2012", 'test'),
-
+        "voc_2007_train": (
+            "voc/VOC2007/JPEGImages",
+            "voc/VOC2007/Annotations/pascal_train2007.json"),
+        "voc_2007_val": (
+            "voc/VOC2007/JPEGImages",
+            "voc/VOC2007/Annotations/pascal_val2007.json"),
+        "voc_2007_test": (
+            "voc/VOC2007/JPEGImages",
+            "voc/VOC2007/Annotations/pascal_test2007.json"),
+        "voc_2012_train": (
+            "voc/VOC2012/JPEGImages",
+            "voc/VOC2012/Annotations/pascal_train2012.json"),
+        "voc_2012_val": (
+            "voc/VOC2012/JPEGImages",
+            "voc/VOC2012/Annotations/pascal_val2012.json"),
+        "voc_2012_test": (
+            "voc/VOC2012/JPEGImages",
+            "voc/VOC2012/Annotations/pascal_test2012.json"),
+        "cityscapes_fine_instanceonly_seg_train": (
+            "cityscapes/images",
+            "cityscapes/annotations/instancesonly_filtered_gtFine_train.json"),
+        "cityscapes_fine_instanceonly_seg_val": (
+            "cityscapes/images",
+            "cityscapes/annotations/instancesonly_filtered_gtFine_val.json"),
+        "cityscapes_fine_instanceonly_seg_test": (
+            "cityscapes/images",
+            "cityscapes/annotations/instancesonly_filtered_gtFine_test.json")
     }
 
     @staticmethod
     def get(name):
-        if "coco" in name:
+        if "coco" in name or cfg.DATASETS.FORCE_USE_JSON_ANNOTATION:
```

Comment:
I'd rather not have this option, and instead have the name tag be representative of what the dataset is or how it should be read.